libs/template/spec/let/let.directive.template-binding.no-complete.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { Component } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { LetDirective } from '@rx-angular/template';
+import { EMPTY, Observable, of } from 'rxjs';
+
+@Component({
+  template: `
+    <ng-container *rxLet="value$; let value; suspense: suspense; error: error">{{
+      value === undefined
+        ? 'undefined'
+        : value === null
+        ? 'null'
+        : (value | json)
+    }}</ng-container>
```

Comment:
Maybe its useful to use the `let e = error; let c = complete;` ?

Reply:
I don't know, in what way would it be useful?

Reply:
In an error case without template to see that the error value is set? 
I guess we have that in another test already. So not that important